buddyboss/buddyboss-platform
# Fix - On Photos tab - text & dropdown is overriding

### All Submissions:


Fixes #2488 

### How to test the changes in this Pull Request:

1. Go to 'Profile > Photos tab'
2. Click on 'Album > Select any album > Upload any image in the album'
3. After uploading an image, please change the tab https://prnt.sc/15citel from "Upload" to "Select Photo" and "Select Photo" to "Upload"
4. Check UI

### Proof Screenshots or Video
https://ibb.co/G0n7tYz

### Other information:
